arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -186,7 +186,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The scope for the lock. "
+            "description": "The scope for the lock. A scope can be defined at subscription, resource group, or resource level. It looks like this: /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{parentResourcePathIfPresent}/{resourceType}/{resourceName}"
```

Comment:
Thanks! Are these other scope strings also valid? Extract below taken from policy management documentation:
"When providing a scope for the assignment, use '/subscriptions/{subscription-id}/' for subscriptions, '/subscriptions/{subscription-id}/resourceGroups/{resource-group-name}' for resource groups, and '/subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/{resource-provider-namespace}/{resource-type}/{resource-name}' for resources."